eventarc/pubsub/README.md

```diff
@@ -16,7 +16,7 @@ gcloud run deploy eventarc-pubsub \
 Create a Cloud Pub/Sub trigger:
 
 ```sh
-gcloud beta events triggers create pubsub-trigger \
+gcloud beta eventarc triggers create pubsub-trigger \
```

Comment:
We can also remove the 'beta' now

Reply:
I don't think so - the filtering part is still beta, as far as I can tell.

Reply:
`gcloud eventarc triggers create` is definetely not beta: https://cloud.google.com/eventarc/docs/creating-triggers

Reply:
But the matching-criteria flag is. Put it this way: I tried it this morning without beta, and it failed. With beta, it works.

Perhaps it should use a different flag instead of matching-criteria?

Reply:
Ah, yes, `matching-criteria` was renamed to `event-filters` in GA. I added this README before we had Eventarc docs. Now that we have Eventarc docs, maybe it's best to remove it? I doubt we'll be able to keep it up to date with all the changes, as evidenced here.